robots/preservation_ingest/update_catalog.rb

```diff
@@ -1,10 +1,13 @@
+require 'faraday'
+require 'retries'
+
 # Robot package to run under multiplexing infrastructure
 module Robots
   # Use DorRepo/SdrRepo to match the workflow repo (and avoid name collision with Dor module)
   module SdrRepo
     # The workflow package name - match the actual workflow name, minus ending WF (using CamelCase)
     module PreservationIngest
-      # what does this class do? FIXME
+      # Sends POST or PATCH request to create or update object
```

Comment:
How about more clarity on which object is being updated, where?  Maybe you can improve on:

`# sends ReST Create or Update Version request to PreservationCatalog, depending on whether this is a new Moab or an update to an existing Moab`